integration-tests/tests/src/typings.d.ts

```diff
@@ -153,6 +157,11 @@ type RealmContext = {
    */
   closeRealm(options?: Partial<CloseRealmOptions>): Promise<void>;
 } & Mocha.Context;
+type MultiRealmContext = {
+  openedInfo: { realm: Realm; config: Realm.Configuration }[];
+  getRealm: (config: any) => Promise<Realm>; //any should be OpenRealmConfiguration
```

Comment:
I think I can't use `OpenRealmConfiguration` here because it would create a circular reference and TS didn't seem to appreciate that